webapp/src/pages/Dashboard.js

```diff
@@ -1,15 +1,45 @@
+import api from "../api";
+import scooterIcon from "../assets/images/kick-scooter.png";
 import Header from "../components/Header";
 import signOut from "../modules/signOut";
 import { useUser } from "../state/useUser";
 import React from "react";
+import Badge from "react-bootstrap/Badge";
 import Button from "react-bootstrap/Button";
 import Col from "react-bootstrap/Col";
 import Container from "react-bootstrap/Container";
+import Navbar from "react-bootstrap/Navbar";
 import Row from "react-bootstrap/Row";
+import Table from "react-bootstrap/Table";
 import { Link } from "react-router-dom";
 
 const Dashboard = () => {
   const { user } = useUser();
+  const [savingsBalance, setSavingsBalance] = React.useState("");
+  const [accountBalance, setAccountBalance] = React.useState("");
+  const [ledgers, setLedgers] = React.useState([]);
+
+  React.useEffect(() => {
+    api.dashboard().then((r) => {
+      setSavingsBalance(r.data.lifetimeSavings.cents / 100.0);
+      setAccountBalance(r.data.paymentAccountBalance.cents / 100.0);
+      const getLast60Days = (ledgers) => {
```

Comment:
This sort of thing should always be on the backend. It is a red flag if you need things like this on the frontend, so bring it up when you notice you are doing it.